Resources/Prototypes/Entities/Structures/Power/apc.yml

```
@@ -93,6 +93,62 @@
         acts: [ "Destruction" ]
   - type: StationInfiniteBatteryTarget
 
+# APC under construction
+- type: entity
+  noSpawn: true
+  id: APCFrame
+  name: APC Frame
```

According to the conventions, names should be lowercase unless it's an acronym. https://docs.spacestation14.io/en/getting-started/conventions
```suggestion
  name: APC frame
```